zmon-controller-app/src/main/java/org/zalando/zmon/service/impl/ZMonServiceImpl.java

```diff
@@ -113,6 +113,8 @@ public ExecutionStatus getStatus() {
         final Map<String, Response<String>> lastUpdate = new HashMap<>();
         final Map<String, Response<String>> invocations = new HashMap<>();
 
+        ServiceLevelStatus.ServiceLevelStatusData serviceLevelStatus = new ServiceLevelStatus.ServiceLevelStatusData();
```

Comment:
Why do you need to create an empty `ServiceLevelStatusData` here? It's created in line 650 anyway